softdevice_handler: fix issue in logging SoftDevice version (#828)
nrfconnect/sdk-nrf-bm: doc/nrf-bm/release_notes/release_notes_changelog.rst

SoftDevice Handler
==================

> * Fixed a bug where using the :kconfig:option:`CONFIG_NRF_SDH_LOG_SD_INFO` Kconfig option in MCUboot board targets would log invalid SoftDevice version data.

Reviewer: ```suggestion
* Fixed an issue where using the :kconfig:option:`CONFIG_NRF_SDH_LOG_SD_INFO` Kconfig option for MCUboot board targets would log invalid SoftDevice version data.
```

>   The logging now takes into account for the SoftDevice partition offset in the MCUboot board targets.

Reviewer: ```suggestion
  The logging now takes the SoftDevice partition offset into account for those board targets.
```
